README.md

```diff
@@ -29,3 +29,34 @@ Storage layer with super easy to use in-memory data views and Sherlock-based per
 
 #### [Type Evolution](https://github.com/C5T/Current/blob/master/TypeSystem/Evolution.md)
 Compact and autogenerated C++ framework to evolve objects from one type hierarchy into another type hierarchy while remaining fully within the strong typing paradigm.
+
+## Contribution
+
+Welcome, contributors! Please [start here by signing the CLA](https://github.com/C5T/Current/blob/master/contributors/README.md).
+
+## Quick Start
+
+Install the development dependencies (see below).
+
+Clean, build and run all tests, verify code:
```

Comment:
"and run tests" or "and run all the tests". Not sure what does "verify code" mean here.